core/tauri/src/manager.rs

```diff
@@ -627,6 +627,10 @@ impl<R: Runtime> WindowManager<R> {
     })
   }
 
+  pub fn asset_iter(&self) -> Box<dyn Iterator<Item = (&&str, &&[u8])> + '_> {
+    self.inner.assets.iter()
+  }
```

Comment:
I think we don't need to expose this one here, the `AssetResolver::iter` is enough

Reply:
this is private, it's just a helper otherwise we need to make more Manager fields public

Reply:
right, I though this was on the `Manager` trait, this is fine then.